docs/examples/aircraft_pairing.rst

```diff
@@ -0,0 +1,158 @@
+600 s and only pair 1-2 flight days to reduce memory constraints and run a 
+jupyter notebook. This Jupyter notebook 
+``examples/jupyter_notebooks/AEROMMA_UFS-AQM_Aircraft_Pairing.ipynb``
+pairs `AEROMMA <https://csl.noaa.gov/projects/aeromma/>`_ observational data with 
+the UFS-AQM model with a long resampling window of 600 s for testing purposes. This 
+Jupyter notebook calls this YAML file ``examples/yaml/control_aircraft_looping_AEROMMA_UFSAQM.yaml``,
+which describes how to perform the analysis. This supplementary YAML file 
+``examples/yaml/supplementary_yaml/supplementary_aircraft_looping_file_pairs_AEROMMA_UFSAQM.yaml``,
+is also optionally called to describe which observations and model data files should be combined.
+
+When you are ready for full analysis, you can reduce this resampling window (e.g., 30 s), 
+update to pair over all days of a campaign, and submit a job on your linux machine like 
+the example below, which will pair `AEROMMA <https://csl.noaa.gov/projects/aeromma/>`_ 
+observational data with the UFS-AQM model and runs on the NOAA Hera HPC system. Submitting a job 
+for pairing aircraft data is much faster, so this is the recommended approach except for testing 
+or debugging.
+
+This python script ``examples/submit_jobs/run_aircraft_pairing_loop_AEROMMA.py`` 
+pairs `AEROMMA <https://csl.noaa.gov/projects/aeromma/>`_ observational data with 
+the UFS-AQM model with a short resampling window of 30 s for full analysis. This 
+python script calls this YAML file ``examples/yaml/control_aircraft_looping_AEROMMA_UFSAQM-submit.yaml``,
+which describes how to perform the analysis. This supplementary YAML file 
+``examples/yaml/supplementary_yaml/supplementary_aircraft_looping_file_pairs_AEROMMA_UFSAQM-submit.yaml``,
+is called to describe which observations and model data files should be combined.
+This bash script ``examples/submit_jobs/submit_hera_aircraft_AEROMMA.sh`` is an example for 
+how to run this python script on Hera (NOAA's HPC) and can be easily adapted to other computers as well.
+
+
+Steps for Pairing Procedure
+---------------------------
+
+Submitting a python script job or using jupyter notebook processes the data in the same way 
+with the following steps.
+
+1) We import the ``loop_pairing`` function from ``melodies_monet.util.tools``
+
+2) We read in a control file that explains all the pairing parameters
+
+3) There are two options for providing the model and observation data for pairing
+
+	* Option 1) Provides the info in a dictionary
+
+	* Option 2) Provide the info in a supplementary yaml file. This option is 
+	  specifically useful when submitting a job for the analysis as then you do not 
+	  have to update the python script.
+
+To support creating a dictionary or supplementary yaml file to determine the pairing, 
+we have also created a function to find the time bounds in the observation file. To use 
+this, first import the ``find_obs_time_bounds`` function from ``melodies_monet.util.tools``. 
```

Comment:
You could trying actually linking to it, `` :func:`melodies_monet.util.tools.find_obs_time_bounds` ``, see if it works.

Reply:
Same for `loop_pairing`